src/partials/recentBlocks/recentBlocks.vue

```diff
@@ -3,38 +3,49 @@
     <div class="inner">
       <h2>Recent generations</h2>
       <p>View the latest generations on the aeternity blockchain</p>
-      <table v-if="generations.length">
-        <tr v-for='(b, i) in generations' :key="i">
+      <table>
+        <tr v-for='i in 3' :key="i">
```

Comment:
Why don't cycle through elements instead of indexes, as it was before?

Reply:
RecentBlocks usually show the last three mined blocks.

Reply:
@davidyuk We need to show placeholders at place of elements,and they will shown before the elements are loaded, so there's nothing to cycle through at that time.

Reply:
I think it is clearer to add an empty elements: 006a21ce13450c3806ffbf7283f45cc1fad177ec